omnibus/config/projects/inspec.rb

```diff
@@ -17,21 +17,52 @@
 #
 
 name 'inspec'
+friendly_name "Inspec"
 maintainer 'Chef Software, Inc <maintainers@chef.io>'
 homepage 'https://github.com/chef/inspec'
 
 license 'Apache-2.0'
 license_file '../LICENSE'
 
-# Defaults to C:/inspec on Windows
-# and /opt/inspec on all other platforms
-install_dir "#{default_root}/#{name}"
+# Defaults to C:/opscode/inspec on Windows
+# and /opt/inspec on all other platforms.
+if windows?
+  install_dir "#{default_root}/opscode/#{name}"
```

Comment:
Yep - everything on windows is under opscode.  C:\opscode corresponds to /opt.  C:\chef corresponds to /etc/chef.  Changing that is pretty much impossible at this point between all the chef products we have making that assumptions :(